posthog/clickhouse/cluster.py

```diff
@@ -79,11 +79,11 @@ def result(
 
 
 class ConnectionInfo(NamedTuple):
-    address: str
+    host_name: str
```

Comment:
I suppose it doesn't actually matter here whether this is a name or address, this could just be `host` and accurately describe either one